networks: update seeds and checkpoint for 5.x

pinheadmz opened the pull request:
merge to master, then backport to 5.x branches

rithvikvibhu commented:
Seeds:
- `138.197.52.107` to be removed (mine, marked as deprecated last time, now offline)
- `172.104.177.177` one of @chjj's seems to be down

(add) More reachable nodes found with hnsnodes:
```
89.163.154.217
139.178.67.226
47.242.86.29
99.92.204.3

# Hosted on flux, requires port:
23.245.194.179:39001
65.108.135.176:39001
76.69.97.54:39001
216.8.155.190:39001
31.7.195.41:39001
88.112.71.231:39001
103.224.165.120:39001
161.97.117.139:39001
66.206.29.25:39001
24.199.233.98:39001
65.108.21.91:39001
95.216.118.230:39001
```

(remove) Nodes not reachable / hnsnodes didn't find:
```
81.6.58.121
185.232.71.108
137.184.156.222
176.96.136.143
150.136.231.120
```

<details>
  <summary>raw dump from hnsnodes</summary>

```json
[["23.245.194.179", 39001, 3, 125587, "/hsd:3.0.1/"], ["89.163.154.217", 12038, 3, 151668, "/hsd:4.0.2/ape.c/"], ["74.207.247.120", 12038, 1, 151668, "/hsd:2.4.0/"], ["129.153.166.244", 12038, 3, 151668, "/hsd:4.0.2/djlederle/"], ["108.61.206.38", 12038, 3, 151668, "agnaehost"], ["98.47.90.111", 12038, 1, 151668, "/hsd:4.0.2/"], ["66.29.151.42", 12038, 3, 151668, "/hsd:4.0.0/"], ["45.79.134.225", 12038, 1, 151668, "/hsd:2.4.0/"], ["140.238.196.136", 12038, 3, 151668, "/hsd:4.0.1/steve/"], ["89.58.17.86", 12038, 3, 151668, "/hsd:4.0.0/ape.c/"], ["178.18.254.92", 12038, 3, 151668, "/hsd:4.0.0/ape.c/"], ["5.161.64.49", 12038, 3, 151668, "/hsd:4.0.0/"], ["82.223.165.215", 12038, 3, 151668, "/hsd:4.0.0/ape.c/"], ["173.255.209.126", 12038, 1, 151668, "/hsd:2.4.0/"], ["140.238.164.155", 12038, 3, 151668, "/hsd:4.0.2/"], ["139.178.67.226", 12038, 3, 151668, "/hsd:3.0.1/"], ["154.26.133.109", 12038, 3, 151668, "/hsd:4.0.0/ape.c/"], ["205.250.112.133", 12038, 3, 151668, "/hsd:4.0.2/"], ["168.138.31.136", 12038, 3, 151668, "/hsd:4.0.2/HNSAU/"], ["165.22.151.242", 12038, 3, 151668, "/hsd:4.0.0/easyhandshake.com/"], ["65.108.135.176", 39001, 3, 151668, "/hsd:3.0.1/"], ["47.242.86.29", 12038, 1, 151668, "/hsd:2.3.0/"], ["107.152.32.250", 12038, 3, 151668, "/hsd:3.0.1/"], ["139.59.211.187", 12038, 1, 151668, "/hsd:4.99.0/"], ["76.69.97.54", 39001, 3, 151668, "/hsd:3.0.1/"], ["129.153.177.220", 12038, 3, 151668, "/hsd:4.0.2/"], ["202.61.201.15", 12038, 3, 151668, "/hsd:3.0.1/"], ["107.152.33.71", 12038, 3, 151668, "/hsd:3.0.1/"], ["66.94.118.127", 12038, 3, 151668, "/hsd:4.0.0/ape.c/"], ["99.92.204.3", 12038, 3, 151668, "/hsd:4.99.0/"], ["193.41.237.153", 12038, 3, 151668, "/hsd:4.0.2/ape.c/"], ["149.102.152.193", 12038, 3, 151668, "/hsd:4.0.0/ape.c/"], ["172.104.214.189", 12038, 1, 151668, "/hsd:2.4.0/"], ["44.229.138.206", 12038, 3, 151668, "/hsd:2.1.5/"], ["104.254.246.153", 12038, 3, 151668, "/hsd:4.99.0/"], ["216.8.155.190", 39001, 3, 151668, "/hsd:3.0.1/"], ["204.99.129.8", 12038, 3, 151668, "/hsd:4.0.2/"], ["31.7.195.41", 39001, 3, 151668, "/hsd:3.0.1/"], ["159.69.46.23", 12038, 3, 151668, "/hsd:4.0.0/"], ["108.175.4.26", 12038, 3, 151668, "/hsd:4.99.0/"], ["76.217.158.15", 12038, 3, 151668, "/hsd:4.0.2/"], ["54.184.104.94", 12038, 3, 151668, "/hsd:2.2.0/"], ["207.244.233.76", 12038, 3, 151668, "/hsd:4.0.0/ape.c/"], ["47.108.194.84", 12038, 1, 151668, "/hsd:2.3.0/"], ["88.112.71.231", 39001, 3, 127668, "/hsd:3.0.1/"], ["85.214.33.200", 12038, 3, 151668, "/hsd:4.0.0/another.software/"], ["139.162.183.168", 12038, 1, 151668, "/hsd:2.4.0/"], ["147.124.228.198", 12038, 1, 151668, "/hsd:4.0.0/another.software/"], ["103.224.165.120", 39001, 3, 151668, "/hsd:3.0.1/"], ["73.185.40.66", 12038, 3, 151668, "/hsd:4.0.2/"], ["161.97.117.139", 39001, 3, 151668, "/hsd:3.0.1/"], ["66.206.29.25", 39001, 3, 151668, "/hsd:3.0.1/"], ["81.169.159.30", 12038, 3, 151668, "/hsd:4.0.0-rc.1/node-5c61.another.software/"], ["165.227.93.117", 12038, 3, 151668, "/hsd:4.0.0/"], ["24.199.233.98", 39001, 3, 68441, "/hsd:3.0.1/"], ["65.108.21.91", 39001, 3, 103150, "/hsd:3.0.1/"], ["95.216.118.230", 39001, 3, 151668, "/hsd:3.0.1/"], ["212.227.68.123", 12038, 3, 151668, "/hsd:4.0.0/another.software/"], ["152.69.186.119", 12038, 3, 151668, "/hsd:4.0.2/"], ["50.116.35.178", 12038, 3, 151668, "/hsd:4.0.2/sinpapeles/"], ["45.79.95.228", 12038, 3, 151668, "/hsd:4.99.0/"]]
```

</details>

Hopefully by next hsd release, there'll be a webpage to just copy and paste the list in here.

pinheadmz commented:
@rithvikvibhu thanks, squashed those extra hosts in there and removed your old one. Also changed the checkpoint height to 130000, which gives a solid 4-5 months of PoW. That will be the hnsd checkpoint height as well
